packages/react-reconciler/src/__tests__/ReactLazy-test.internal.js

```diff
@@ -879,7 +879,13 @@ describe('ReactLazy', () => {
       },
     );
 
-    expect(Scheduler).toFlushAndYield(['Started loading', 'Loading...']);
+    if (__DEV__) {
+      // Getting the name for the warning cause the loading to start early.
```

Comment:
Don't know how I feel about this. Can you motivate?

Reply:
I think we really should be refactoring the getComponentName stuff. One todo I have is to pass it a fiber because the type is not really enough. If we had the fiber we could read the .type off of that.

It’s called everywhere though so it’s some work to properly clean that up.

What annoys be a bit more is that we have that logic in prod. I suspect that’s what leads this PR to increase bundle size but just a little bit.

We probably should fork getComponentName and accompanying logic into specific DEV and prod versions for different use cases.

Regardless, I’m confident the fix is in the logging and not compromising the logic for lazy resolution.